src/components/modal/iframe/modal.jsx

```diff
@@ -21,11 +20,7 @@ var IframeModal = React.createClass({
             this.props.className
         );
         return (
-            <Modal
-                isOpen={this.props.isOpen}
-                onRequestClose={this.props.onRequestClose}
-                className={this.props.className}
-            >
+            <Modal {...omit(this.props, ['ref', 'src'])}>
```

Comment:
You can exclude `ref` because `ref` is a special attribute, it's not passed in as a prop.